engine/src/main/kotlin/world/gregs/voidps/engine/entity/character/player/Player.kt

```diff
@@ -47,6 +48,7 @@ class Player(
     var accountName: String = "",
     var passwordHash: String = "",
     val body: BodyParts = BodyParts(),
+    var follower: NPC? = null,
```

Comment:
Variables can be used for this, a custom var storing the follower id will be needed, it's also better to store the npc index and look-up the NPC rather than store the NPC directly to prevent accidentally using (and the garbage collector not being able to delete) npcs that no-longer exist
e.g.

```kotlin
val Player.follower: NPC
    get() {
        val index = this["follower_index", -1]
        get<NPCS>.get(index)
    }
```